docs/conf/na/2017/workshops.rst

```diff
@@ -0,0 +1,33 @@
+:template: 2017/na-content.html
+
+Workshops
+=========
+
+This year we are launching workshops, a new activity type inspired once again by our friends at various open-source software conferences. The workshops will be held during the writing day, in dedicated rooms in the conference venue.
+
+The main goal of the workshops is to bring together folks who are interested to learn from different experts in relevant fields in a focused environment.
+
+What can I learn in the workshops?
+----------------------------------
+
+This year we have 2 workshops that cover one writing-related topic and one development-related topic. This is a part of our efforts to cross-train the different roles that our community members encompass.
```

Comment:
> This year we have two workshops, one that covers writing related topics and one that covers development related topics.

^ reads better to me. Also, calling git development related is a stretch I think. Development AND docs as code? :-p :-p